sn_client/src/error.rs

```diff
@@ -46,6 +46,10 @@ pub enum Error {
     #[error("Client received no network activity for {0:?}")]
     InactiveClient(Duration),
 
+    /// Client received an unexpected event.
+    #[error("Client received an unexpected event")]
```

Comment:
Can we not handle this event? Why not? This seems a bit too catch all..

Reply:
That's just being too `safe`, can be removed actually